pkg/apis/acme/v1alpha3/types_issuer.go

```diff
@@ -81,14 +102,25 @@ const (
 	HS512 HMACKeyAlgorithm = "HS512"
 )
 
+// Configures an issuer to solve challenges using the specified options.
+// Only one of HTTP01 or DNS01 may be provided.
 type ACMEChallengeSolver struct {
 	// Selector selects a set of DNSNames on the Certificate resource that
 	// should be solved using this challenge solver.
+	// If not specified, the solver will be treated as the 'default' solver
+	// with the lowest priority, i.e. if any other solver has a more specific
+	// match, it will be used instead.
```

Comment:
This took me a while to understand this. I think it would be worth describe this solver priority on the `ACMEIssuer > Solvers` field as well or instead.